src/clojurewerkz/elastisch/native/conversion.clj

```diff
@@ -1448,3 +1450,70 @@
 (defn ^DeleteIndexTemplateRequest ->delete-index-template-request
   [template-name]
   (DeleteIndexTemplateRequest. template-name))
+
+;;
+;; Bulk Responses
+;;
+
+(defn bulk-item-response->map
+  [^BulkItemResponse item]
+  (let [res {:index (.getIndex item)
+             :_index (.getIndex item)
+             :type (.getType item)
+             :_type (.getType item)
+             :id (.getId item)
+             :_id (.getId item)
+             :version (.getVersion item)
+             :_version (.getVersion item)
+             :op-type (.getOpType item)
+             :failed? (.isFailed item)}]
+    (if (:failed? res)
+      (assoc res :failure-message (.getFailureMessage item))
+      res)))
+
+(defn bulk-response->map
+  [^BulkResponse response]
+  {:took (.getTookInMillis response)
+   :has-failures? (.hasFailures response)
+   :items (mapv bulk-item-response->map (.getItems response))})
+
+(defn remove-underscores [opts]
```

Comment:
Please use a consistent code style: argument vector should be on the next line. Functions written with this style are asking for not having docs.